packages/server/lib/routes.ts

```diff
@@ -299,7 +301,7 @@ router.use(web);
 const webappBuildPath = '../../../webapp/build';
 const staticSite = express.Router();
 staticSite.use('/assets', express.static(path.join(dirname(), webappBuildPath), { immutable: true, maxAge: '1y' }));
-staticSite.use(express.static(path.join(dirname(), webappBuildPath), { setHeaders: () => ({ 'Cache-Control': 'no-cache, private' }) }));
+staticSite.use(express.static(path.join(dirname(), webappBuildPath), { cacheControl: true, maxAge: '1h' }));
```

Comment:
If we don't hash static files for each deploy or prune the cache, there is a chance the browser would use stale content. Might not be an issue though

Reply:
yeah that's why I looked at the files, the only ones left are logo I'll probably integrate them in the code directly later. Worst case it's cached for one hour